src/components/playback/mediasession.js

```diff
@@ -99,15 +105,26 @@ define(['playbackManager', 'nowPlayingHelper', 'events', 'connectionManager'], f
             return;
         }
 
+        // dummy this up
```

Comment:
well if you are asking if i wrote it, no. The actual meaning.. searching for it in google, i found this: https://www.urbandictionary.com/define.php?term=Dummy-up

There is an implementation to prevent notifications like hell to upade mediaSession info. So this should belong to that implementation. I don't know. I could remove the comment.

Reply:
I was wondering what this thing does and what this comment means...

Reply:
Well the code transforms init events into timeupdate for the function context. Then what i see is if event type if timeupdate and there as an update with an interval lesser than 5 seconds, then ignore the player state update. This code was in jf-android initially, but not in jf-web, but i found traces of it in jf-web, like the lastUpdateTime defined, but never used. I think it makes sense to not update very frequently the mediasession if event is timeupdate. If we find some issue with the refresh rate of the sticky notification, we should investigate further, and try to decrease the time between updates.

Reply:
I got the code about time updates.

So after your explanation I propose to remove `// dummy this up` and add a comment like `// transform "init" event into "timeupdate" within this function context` inside the `if()`.

Reply:
sure np, will do

Reply:
it's done